crates/language_model/src/request.rs

```diff
@@ -259,7 +260,7 @@ impl LanguageModelRequest {
                     },
                 })
                 .collect(),
-            stream: true,
+            stream,
             stop: self.stop,
```

Comment:
`o1` models do not support `stop` at the moment.
Maybe the following is a possible solution:
```rust
pub fn into_open_ai(self, model: String, max_output_tokens: Option<u32>) -> open_ai::Request {
    let stream = !model.starts_with("o1-");
    let stop = if model.starts_with("o1-") {
        None
    } else {
        self.stop
    };

    open_ai::Request {
        model,
        messages: self
            .messages
            .into_iter()
            .map(|msg| match msg.role {
                Role::User => open_ai::RequestMessage::User {
                    content: msg.string_contents(),
                },
                Role::Assistant => open_ai::RequestMessage::Assistant {
                    content: Some(msg.string_contents()),
                    tool_calls: Vec::new(),
                },
                Role::System => open_ai::RequestMessage::System {
                    content: msg.string_contents(),
                },
            })
            .collect(),
        stream,
        stop,
        temperature: self.temperature,
        max_tokens: max_output_tokens,
        tools: Vec::new(),
        tool_choice: None,
    }
}
```